Server/Services/adminServiceHandler.py

```diff
@@ -0,0 +1,107 @@
+from DB_Connection.adminQueries import AdminQuery
+from decimal import Decimal
+import pymysql
+
+class AdminServiceHandler:

```

Comment:
Including Both Service and Handler in the name sounds redundant

Reply:
I have changed all the names containing ServiceHandler to Service.